### PR TITLE
Add total spend widget

### PR DESCRIPTION
**Pull request by Horay**
When merged this PR will add a total spend widget

### CODE REVIEW

File: src/components/box/Box.tsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ type BoxProps = {
   backgroundColor?: string;
   radius?: "small" | "medium" | "large";
   spacing?:
+    | "spacing00"
```
Comment (khola) [inline src/components/box/Box.tsx:16]: GNUI Figma says nothing about "spacing0". This creates a mess.

**Review by khola — CHANGES_REQUESTED** [review]

File: src/components/box/Box.tsx (hunk 1 above)
Comment (Horay) [reply src/components/box/Box.tsx]: ok i have changed it

File: src/components/text/Text.tsx [hunk 2]
```diff
@@ -17,10 +17,10 @@ interface TextProps {
 
 const changeTag = (tag: string) => {
   let spacing: string;
-  if(tag !== "p") {
-    spacing = theme.spacing.spacing00
+  if (tag !== "p") {
+    spacing = theme.spacing.spacing00;
```
Comment (khola) [inline src/components/text/Text.tsx:21]: I still see spacing00 here

**Review by khola — CHANGES_REQUESTED** [review]

File: src/components/text/Text.tsx (hunk 2 above)
Comment (Horay) [reply src/components/text/Text.tsx]: Fixed

**Review by khola — APPROVED** [review]

**Review by janosio-nordcloud — APPROVED** [review]

File: src/components/text/Text.tsx [hunk 3]
```diff
@@ -17,10 +17,10 @@ interface TextProps {
...
   let spacing: string;
-  if(tag !== "p") {
-    spacing = theme.spacing.spacing00
+  if (tag !== "p") {
+    spacing = "0";
   } else {
-    spacing = theme.spacing.spacing04
+    spacing = theme.spacing.spacing04;
   }
   return spacing;
```
Comment (nowyDEV) [inline src/components/text/Text.tsx:25]: I would use ternary here
`return tag === "p" ? theme.spacing.spacing04 : theme.spacing.spacing00`

File: src/components/piechart/PieChart.tsx [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+    css`
+      transform: rotate(-90deg) translateX(-${size}px);
+    `}
+`;
+
+export const PieChart: FunctionComponent<IPieChart & SpaceProps> = ({
+  size,
+  progress,
+  strokeWidth,
+  color,
```
Comment (nowyDEV) [inline src/components/piechart/PieChart.tsx:107]: I think it's a good practice to specify the default value here, since the prop is typed as optional.

**Review by nowyDEV — APPROVED** [review]

**Pull request merged**